### PR TITLE
kintsugi: Handle expected errors gracefully.

### PR DESCRIPTION
**Pull request by byohay**
Instead of printing the stack trace, just print the errors.

### CODE REVIEW

File: lib/kintsugi/error.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2021 Lightricks. All rights reserved.
+# Created by Ben Yohay.
+# frozen_string_literal: true
+
+module Kintsugi
+  class EnvironmentError < RuntimeError
```
Comment (ashdnazg) [inline lib/kintsugi/error.rb:6]: document

File: lib/kintsugi/error.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  end
+
+  class MergeError < RuntimeError
```
Comment (ashdnazg) [inline lib/kintsugi/error.rb:9]: document

**Review by ashdnazg — CHANGES_REQUESTED** [review]

File: lib/kintsugi/error.rb (hunk 1 above)
Comment (byohay) [reply lib/kintsugi/error.rb]: Removed `EnvironmentError`

**Comment by byohay** [comment]
@ashdnazg squashed.

File: lib/kintsugi.rb [hunk 3]
```diff
@@ -23,9 +24,11 @@ class << self
     # @raise [ArgumentError]
     #        If the file extension is not `pbxproj` or the file doesn't exist
     #
-    # @raise [RuntimeError]
-    #        If no rebase, cherry-pick, or merge is in progress, or the project file couldn't be
-    #        opened, or there was an error applying the change to the project.
+    # @raise [EnvironmentError]
```
Comment (ashdnazg) [inline lib/kintsugi.rb:27]:
```suggestion
    # @raise [ArgumentError]
```

**Review by ashdnazg — APPROVED** [review]
F&S&M

**Pull request merged**